File: Inventarization.Api/Inventorisation/ViewModels/UserInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Security.Principal;
using System.Web;

namespace Inventorization.Api.ViewModels
{
    public class UserInfo
    {
        public string Token;
        public string Error;
        public string FullName;
        public bool IsAuthorized;
        public string Username;
        public string Password;
        public List<Business.Model.Company> Companies;
        public Business.Model.Company Company;
        public Business.Model.Inventorization DefaultInventorization;
    }
}

File: Inventarization.Android.Client/app/src/main/java/inventory/aeco/network/models/UserInfo.java
package inventory.aeco.network.models;

/**
 * Created by Барашики on 10.06.2017.
 */



public class UserInfo
{
    public String Token;
    public String Error;
    public String FullName;
    public Inventorization[] Inventorizations;
    public boolean IsAuthorized;
    public String Username;
    public String Password;
    public Company[] Companies;
    public Company Company;
    public Inventorization DefaultInventorization;
}
